apps/web/src/hooks/useRegisterNameCallback.ts

```diff
@@ -35,32 +34,10 @@ export function useRegisterNameCallback(
   discountKey?: `0x${string}`,
   validationData?: `0x${string}`,
 ): UseRegisterNameCallbackReturnValue {
-  const { address, chainId, isConnected } = useAccount();
+  const { address } = useAccount();
   const { basenameChain } = useBasenameChain();
-  const {
-    data: callBatchId,
-    writeContractsAsync,
-    isPending: paymasterIsPending,
-    error: paymasterError,
-  } = useWriteContracts();
-  const { data, writeContractAsync, isPending, error } = useWriteContract();
-  const { data: availableCapacities } = useCapabilities({ account: address });
 
-  const capabilities = useMemo(() => {
-    if (!isConnected || !chainId || !availableCapacities) {
-      return {};
-    }
-    const chainCapabilities = availableCapacities[chainId];
-    if (chainCapabilities.paymasterService?.supported) {
-      return {
-        paymasterService: {
-          // url: `${document.location.origin}/api/paymaster`
-          url: 'https://api.developer.coinbase.com/rpc/v1/base-sepolia/1IhTcPOmhK5aEq-4WqRZMJoOh0oPenD2',
-        },
-      };
-    }
```

Comment:
Don't we need this for paymaster support?

Reply:
We do if we're using the `useWriteContracts` experimental hook. But we're working on seeing whether we need to use the hook at all.